milvus_model/dense/jinaai.py

```diff
@@ -52,8 +55,13 @@ def __call__(self, texts: List[str]) -> List[np.array]:
         return self._call_jina_api(texts)
 
     def _call_jina_api(self, texts: List[str]):
+        data = {"input": texts, "model": self.model_name}
+        if self.model_name == "jina-embeddings-v3":
```

Comment:
avoid doing this, just pass them if they are not None, the backend will return an error or not.